docs/standard/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+## ASP.NET 
+
+The original ASP.NET implementation that ships with the .NET Framework.
+
+Sometimes ASP.NET is an umbrella term that refers to both ASP.NET implementations including ASP.NET Core. The meaning that the term carries in any given instance is determined by context. 
+
+See [ASP.NET](/aspnet/#pivot=aspnet).
+
+## ASP.NET Core
+
+A cross-platform, high-performance, open source implementation of ASP.NET built on .NET Core.
+
+See [ASP.NET Core](/aspnet/#pivot=core).
+
+## CLR
+
+Common language runtime.
+
+The exact meaning depends on the context, but this usually refers to the runtime of the .NET Framework. The CLR is a virtual machine that not only executes apps but also generates and compiles code on-the-fly using a JIT compiler. The current Microsoft CLR implementation is Windows only.
+
+## CoreCLR
+
+.NET Core common language runtime.
+
+This CLR is built from the same code base as the CLR. Originally, CoreCLR was the runtime of Silverlight and was designed to run on multiple platforms, specifically Windows and OS X. CoreCLR is now part of .NET Core and represents a simplified version of the CLR. It's still a cross platform runtime. CoreCLR is also a virtual machine with JIT and code execution capabilities.
+
+## CoreFX
+
+.NET Core framework.
+
+Also known as the .NET Core Base Class Library (BCL). A set of  System.* (and to a limited extent  Microsoft.\*) libraries. The BCL is a general purpose, lower-level frarmework that higher-level application frameworks, such as ASP.NET Core, build on. The source code of the .NET Core BCL is contained in the [CoreFX repository](https://github.com/dotnet/corefx). However, the majority of the .NET Core APIs are also available in the .NET Framework, so you can think of CoreFX as a fork of the .NET Framework BCL.
+
+## CoreRT
+
+.NET Core runtime.
+
+In contrast to the CLR/CoreCLR, CoreRT is not a virtual machine, which means it doesn't include the facilities to generate and run code on-the-fly because it doesn't include a [JIT](#jit). It does, however, include the [GC](#gc) and the ability for runtime type identification (RTTI) and reflection. However, its type system is designed so that metadata for reflection isn't required. This enables having an [AOT](#aot) tool chain that can link away superfluous metadata and (more importantly) identify code that the app doesn't use. CoreRT is in development.
+
+See [Intro to .NET Native and CoreRT](https://github.com/dotnet/corert/blob/master/Documentation/intro-to-corert.md)
+
+## ecosystem
+
+All of the runtime software, development tools, and community resources that are used to build and run applications for a given technology.
+
+The term ".NET ecosystem" differs from similar terms such as ".NET stack" in its inclusion of third-party apps and libraries. Following is an example in a sentence:
+
+- "The motivation behind the .NET Standard is establishing greater uniformity in the .NET ecosystem." ([.NET Standard](net-standard.md))
```

Comment:
Since there's only one item, do u still want the bullet? ... No hardcore worries from the style manual, of course, as they only dissuade from having this, but it is a "list" of one item. I have to agree with Mrs. Calvary, my 7th grade English teacher, on this one! lol Also because there's only one item, I think the example sentence could be just placed right on the end of the prior sentence in the paragraph. Just a suggestion.